doc/api/events.md

```diff
@@ -427,8 +426,8 @@ Returns a reference to the `EventEmitter` so calls can be chained.
 * `listener` {Function} The callback function
 
 Adds a **one time** `listener` function for the event named `eventName` to the
-*beginning* of the listeners array. This listener is invoked only the next time
-`eventName` is triggered, after which it is removed.
+*beginning* of the listeners array. This listener is removed, and then invoked
+only the next time `eventName` is triggered.
```

Comment:
To me this sounds a bit like the listener is removed instantly after being added. Maybe just copy the corresponding sentence for `.once()` from above?